src/main/java/de/muenchen/dave/services/messstelle/UnauffaelligeTageService.java

```diff
@@ -19,4 +21,11 @@ public List<UnauffaelligerTag> getUnauffaelligeTageForMessstelle(final String ms
         return unauffaelligeTageRepository.findByMstId(mstId);
     }
 
+    public long countAllUnauffaelligetageByMstIdAndKalendertagDatumGreaterThanEqualAndKalendertagDatumLessThanAndTagestypIn(final String mstId,
```

Comment:
Da mindestens ein Methodenparameter umbricht jeden Paramter in eine Zeile schreiben.

Reply:
Den Namen der Servicemethode weniger Technisch beschrieben:

Vorschlag:

`countAllUnauffaelligeTageForMstIdAndStartAndEndDateAndForTagestypen` oder so.